pages/runtime/bandwidth-reduction.md

```diff
@@ -0,0 +1,44 @@
+# Controlling device's Bandwidth / Data usage.
+
+In order to provide our users with a clear view of their devices, the resin device Agent constantly keeps our API informed about the devices' conditions and applies any changes, like downloading new application updates or environment variable changes by negotiating with our API. While a quick reflection of actions is great during development, it comes at a cost of increased data usage. 
+
+In order to give our power users control over data flow we enabled a few RESIN [environment variables](http://docs.resin.io/#/pages/management/env-vars.md).
```

Comment:
Might be better to specify it's Config Vars (dunno if the UX for this is changing, but the difference is worth noting)